pkg/operator/ceph/cluster/osd/update_test.go

```diff
@@ -144,19 +144,26 @@ func Test_updateExistingOSDs(t *testing.T) {
 	executor = &exectest.MockExecutor{
 		MockExecuteCommandWithOutputFile: func(command string, outFileArg string, args ...string) (string, error) {
 			t.Logf("command: %s %v", command, args)
-			if fmt.Sprintf("%s %s %s", command, args[0], args[1]) != "ceph osd ok-to-stop" {
```

Comment:
The more common pattern we have in the tests is to compare individual args instead of formatting them back together, if you wouldn't mind that pattern. For example:
```go
if args[0] == "osd" && args[1] == "ok-to-stop" {
  ...
}
```